Gems/Atom/RHI/Vulkan/Code/Include/Atom/RHI/VkAllocator.h

```diff
@@ -0,0 +1,31 @@
+/*

```

Comment:
The current pattern is setup where all the header files actually live within C:\GitHub\o3de\Gems\Atom\RHI\Vulkan\Code\Source\RHI. Please move this file to that directory. Thanks.